### PR TITLE
Reformatted unset_default_device #1498

### PR DESCRIPTION
**Pull request by naailkhan28**

### CODE REVIEW

**Review by chie2727 — CHANGES_REQUESTED** [review]
Hi @naailkhan28 ,
It seems you are carrying out the reformatting task for this function, so please continue and carry out steps 2 to 4 as explained in the task description: https://github.com/unifyai/ivy/issues/814

**Comment by naailkhan28** [comment]
> Hi @naailkhan28 , It seems you are carrying out the reformatting task for this function, so please continue and carry out steps 2 to 4 as explained in the task description: #814

I've added a docstring as per the instructions, however this function has no arguments and doesn't return anything so I don't see any way I can add functions arguments, type hints, or docstring examples?

**Comment by chie2727** [comment]
Since the function has no return, you can add to the function argument by indicating that the return value is `None`.
For example:
https://github.com/unifyai/ivy/blob/44d3b9ca52115cf769f2f2b9c7a192ef7bf9de4a/ivy/functional/ivy/device.py#L255

You don't have to worry about type hints since there are no inputs.
For the docstring examples you can create simple examples of how the function is used.

File: ivy/functional/ivy/device.py [hunk 1]
```diff
@@ -551,8 +551,19 @@ def set_default_device(device: Union[ivy.Device, ivy.NativeDevice]):
...
-def unset_default_device():
-    """"""
+def unset_default_device()-> None:
+    """Resets the default device to "cpu".
+
+    Examples
+    --------
+    >>> ivy.set_default_device("gpu:0")
+    >>> ivy.default_device()
+    device(type="gpu:0")
```
Comment (chie2727) [inline ivy/functional/ivy/device.py:561]: Thanks for making the edits!
I just want to check the format of the output you're getting when running this example locally.
Is the format of your output showing as `device(type="gpu:0")` and not just `gpu:0` ?

clip_2

**Comment by naailkhan28** [comment]
Hi, thanks for bringing that up! I actually did get just `gpu:0` when I ran the code locally but I initially wrote my example to match the example for the `clear_mem_on_dev` function. I've updated my docstring example now to reflect the actual output of the functions :)

**Review by chie2727 — APPROVED** [review]

**Comment by chie2727** [comment]
Hi @naailkhan28 , 
Okay, I see! Thanks for making the edits.
I'll merge in your changes now.

**Pull request merged**